rbac/management/role/definer.py

```diff
@@ -75,15 +79,19 @@ def _make_role(data):
     )
     role, created = Role.objects.get_or_create(name=name, defaults=defaults, tenant=public_tenant)
 
+    dual_write_handler = SeedingRelationApiDualWriteHandler()
```

Comment:
Nit: we can put this initialization outside of the function so we don't have to call it for every role

Reply:
Like constructing one instance above, so it can be used for the whole set of roles and passed into _make_role() so its lifetime is processing the batch, or just pulling it out to the module so its lifetime is forever?